docs/code/introduction/variables_in_code.md

```diff
@@ -0,0 +1,42 @@
+# Variables in Code
+
+## Player Variables in Code
+
+Player variables are only accessible when a game is running. Be prepared that the current player may change in a multiplayer game.
```

Comment:
For clarity, "Note that in a multiplayer game "current player" refers to the player whose turn is up. This is the most common use for player variables, but you can also reference a specific player number when you need to."

Reply:
Couldn't figure out how to do it... If you have a hint I can add that. Is a very valid point.